db/migrate/20190214000759_create_speedrun_dot_com_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSpeedrunDotComUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :speedrun_dot_com_users, id: :uuid do |t|
+      t.belongs_to :user, foreign_key: true, null: false, index: {unique: true}
+
+      t.string :srdc_id, null: false, index: {unique: true}
+      t.string :name,    null: false
+      t.string :url,     null: false
+      t.string :api_key, null: false, index: {unique: true}
```

Comment:
Is there a reason to make an index for `api_key`?  I thought those were for searches and it doesn't appear we're doing any lookups by key.